tests/unit/browser/test_history.py

```diff
@@ -229,6 +237,18 @@ def test_exclude(self, web_history, config_stub):
         assert list(web_history)
         assert not list(web_history.completion)
 
+    def test_no_immedate_duplicates(self, web_history, caplog, mock_time, mocker):
```

Comment:
Can't this test just check `list(web_history)` like the rest? I don't see a reason to introduce mocks here.